app/components/search-form.hbs

```diff
@@ -22,7 +22,7 @@
     local-class="input-lg"
     name="q"
     id="cargo-desktop-search"
-    placeholder="Click or press 'S' to search..."
```

Comment:
The "Click or press S" refers to how to get the search field focussed. Since we auto focus it these days this can be a bit misleading 😅

Reply:
It definitely is misleading. How about making it more clear?